base_transaction_id/views/invoice.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+  <record model="ir.ui.view" id="invoice_view_custom">
```

Comment:
`id` must be declared before `model`

Reply:
Please give me a reference on this, according to the codebase in the OCA it doesn't make sense:

``` shell
└─> ack record.*id.*record --xml | wc -l
1765
└─> ack record.*model.*id --xml | wc -l
30860
```

id is almost always after model.

Reply:
@yvaucher https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#format